src/corrgi/corrgi.py

```diff
@@ -25,21 +24,30 @@ def compute_autocorrelation(
     correlation = corr_type(**kwargs)
     correlation.validate([catalog, random])
     counts_dd, counts_rr = compute_autocorrelation_counts(catalog, random, correlation)
-    num_galaxies = catalog.hc_structure.catalog_info.total_rows
+    num_particles = catalog.hc_structure.catalog_info.total_rows
```

Comment:
total_rows is often none unless a catalog has just been loaded from disk, so this might not work for some inputs. (e.g. after a spatial filter)

Reply:
This issue has been captured in https://github.com/lincc-frameworks-mask-incubator/corrgi/issues/9.

Reply:
This issue has been captured in https://github.com/lincc-frameworks-mask-incubator/corrgi/issues/9.